queue/abstract/driver/utubettl.lua

```diff
@@ -290,24 +290,24 @@ function method.take(self)
     end
 end
 
-local function process_neighbour(self, task, operation)
-    self:on_task_change(task, operation)
-    if task ~= nil then
-        local neighbour = self.space.index.utube:min{state.READY, task[i_utube]}
-        if neighbour ~= nil and neighbour[i_status] == state.READY then
-            self:on_task_change(neighbour)
-        end
+local function wake_neighbour(self, utube)
+    local neighbour = self.space.index.utube:select({state.READY, utube}, {limit=1})[1]
```

Comment:
I think using min() with checks is more correct (with link to https://github.com/tarantool/tarantool/issues/3167)

Reply:
aren't they equivalent?

Reply:
I meant that the behavior of max() was different from expected and I see no reason to replace min() to select(). I agree with you that the default iterator is ALL(same as GE) and the results will be equivalent. But generally min () is used in utubettl driver and IMHO: it better reflects the idea. I may be wrong. What is the reason for choosing select()?

Reply:
from the 1.10 docs:
> In a future version of Tarantool, index:min(key value) will return nothing if key value is not equal to a value in the index.

so we cannot be sure that `neighbour` will be from the same utube on tarantool 1.10

Reply:
Hmm, as I understand you say about https://github.com/tarantool/tarantool/commit/8bbd4a6803a77673c7f146b6946dfcb9c814dce4 . Looks like you're right! Please add a comment. If you add a test, it will be great!

Reply:
> I meant that the behavior of max() was different from expected and I see no reason to replace min() to select(). I agree with you that the default iterator is ALL(same as GE) and the results will be equivalent.
The default iterator is 'EQ'.